app/assets/stylesheets/content/_context_menu.lsg

```diff
@@ -1 +1,59 @@
 # Context menu
+
+```
+<div class="action-menu">
```

Comment:
Please add the class `dropdown` here, too. Or add an entry for the dropdown menus.